README.md

```diff
@@ -114,6 +114,18 @@ OK? (y/n) [y]:
 
 See the [strider-cli] for more details.
 
+If you want to use idap authorization, you can set the ldap config like:    
+
+```javascript
+ {
+    "ad_url": ldap://host:port,
+    "ad_baseDN": dnString,
+    "ad_username": username,
+    "ad_password": password,
+    "ad_adminDN": dnString
```

Comment:
We try to name the config items with camelCase. Plus if this is in a seperate file, we can drop the prefix `ad_`, since the file represents ldap.